src/api/services/lectureService.ts

```
@@ -0,0 +1,486 @@
+            weight: cleanWeight || '0%',
+            maxScore,
+            isPublic: cells[3]?.textContent?.includes('공개') || 
+                     cells[3]?.textContent?.includes('O') || 
+                     cells[3]?.textContent?.includes('checked') || false,
+            description: cells[4]?.textContent?.trim() || ''
+          });
+        }
+      });
+      
+    } else {
+    }
```

ESLint

[error] Empty block statement.

(no-empty)